addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/UserControlledOpenRedirectScanner.java

```diff
@@ -79,9 +83,16 @@ private void checkUserControllableLocationHeaderValue(
             return;
         }
 
-        String protocol = null;
+        String requestDomain = null;
+        try {
+            requestDomain = msg.getRequestHeader().getURI().getAuthority();
+        } catch (URIException ex) {
+            LOGGER.error(
+                    "Unable to get URI from Request. Ignoring and moving ahead with the scanning OpenRedirect",
```

Comment:
Should say `...get authority from URI...` instead, also worth including the URI in the message.

Reply:
included the URI in the log.